pkgs/development/python-modules/scikit-image/default.nix

```diff
@@ -43,12 +52,41 @@ buildPythonPackage rec {
 
   checkInputs = [ pytest ];
 
-  # No tests in archive
-  doCheck = false;
+  # See the explanation above
+  checkPhase = ''
+    rm -r skimage
+    export INSTALL_PATH=$out/${python.sitePackages}
+    pytest $INSTALL_PATH \
```

Comment:
Is there a good reason not to use pytestCheckHook? If so, I think that deserves explanation in a comment.

Reply:
The reason is to add correct paths to tests, both the root test path and ignore paths. I will add it to the comments.

Reply:
We should be able to add the argument with pytestFlagsArray and use the other flags to ignore the failing tests.

Reply:
Could it be made when a runtime value is needed? I mean, the `$out/${python.sitePackages}` is known only at runtime, which is needed to specify e.g. test root.

Reply:
> Could it be made when a runtime value is needed?

I don't understand your question. We can add arbitrary things to pytest.

Reply:
But `$out` is only readable as an environment variable (i.e. only known at runtime, whereas most of the values we pass to `pytestCheckHook` are baked into the `.drv`)

Reply:
``${placeholder $out}``?

Reply:
> `${placeholder $out}`?

Sorry, never saw this feature in the manual. I will update the expression accordingly.